test/models/user_test.rb

```diff
@@ -1,7 +1,81 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
```

Comment:
Let's just add tests for when a user is deleted, the associated friendships, requests, posts, comments and likes should also be deleted